signbank/dictionary/forms.py

```diff
@@ -930,12 +763,19 @@ def __init__(self, queryDict, *args, **kwargs):
         languages = kwargs.pop('languages')
         super(LemmaSearchForm, self).__init__(queryDict, *args, **kwargs)
 
+        count_languages = len(languages)
         for language in languages:
             # and for LemmaIdgloss
             lemma_field_name = self.lemma_search_field_prefix + language.language_code_2char
-            setattr(self, lemma_field_name, forms.CharField(label=_("Lemma")+(" (%s)" % language.name)))
+            if count_languages > 1:
+                lemma_label = _("Lemma")+(" (%s)" % language.name)
+            else:
+                lemma_label = _("Lemma")
+            setattr(self, lemma_field_name, forms.CharField(label=lemma_label))
```

Comment:
This could be written as:
```python
lemma_label = _("Lemma")+(" (%s)" % language.name) if count_languages > 1 else _("Lemma")
```

There are more places where something similar could be done.

Reply:
Ouch! Yes, I see. I am in the habit of doing them straight-line because I don't trust that the model translations and `_()` expressions are going to be combined correctly. (I made an issue about this a long time ago (which nobody responded to, hence closed), because I had to flatten out some comprehensions because they didn't work with non-ascii languages. So superstition makes me code them imperatively to force evaluation order.)

Reply:
Okay, I've revised that. It looked hyper-paranoid from me. Like I don't trust the interpreter.